### PR TITLE
Upgrade Ember to version 3.8

### PR DESCRIPTION
**Pull request by Haz901**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,10 +17,10 @@
     "@ember/optional-features": "~0.7.0",
     "babel-eslint": "^10.0.1",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-cli": "~3.7.1",
+    "ember-cli": "~3.8.1",
     "ember-cli-app-version": "^3.2.0",
     "ember-cli-dependency-checker": "^3.1.0",
-    "ember-cli-eslint": "^5.1.0",
+    "ember-cli-eslint": "^5.0.0",
```
Comment (michaelw8) [inline package.json]: Is this downgrade needed?

File: config/ember-try.js [hunk 2]
```diff
@@ -12,27 +12,23 @@ module.exports = function() {
...
-          name: 'ember-lts-2.16',
+          name: 'ember-lts-2.18',
           env:  {
             EMBER_OPTIONAL_FEATURES: JSON.stringify({ 'jquery-integration': true })
           },
           npm: {
             devDependencies: {
-              '@ember/jquery': '^0.6.0',
-              'ember-source':  '~2.16.0'
+              '@ember/jquery': '^0.5.1',
```
Comment (michaelw8) [inline config/ember-try.js]: Downgrade?

File: package.json (hunk 1 above)
Comment (Haz901) [reply package.json]: No, but because of the ^ it just matches the latest 5.x.x, so this makes no difference

File: config/ember-try.js (hunk 2 above)
Comment (Haz901) [reply config/ember-try.js]: Probably came from the blueprint, but this also matches the latest minor version, so it doesn't change anything

**Pull request merged**